core-api/src/main/java/com/optimizely/ab/odp/ODPEvent.java

```diff
@@ -0,0 +1,64 @@
+/**
+ *    Copyright 2022, Optimizely Inc. and contributors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.optimizely.ab.odp;
+
+import java.util.Map;
+
+public class ODPEvent {
+    private String type;
+    private String action;
+    private Map<String, String > identifiers;
+    private Map<String, String> data;
+
+    public ODPEvent(String type, String action, Map<String, String> identifiers, Map<String, String> data) {
```

Comment:
```suggestion
    public ODPEvent(String type, String action, Map<String, String> identifiers, Map<String, Object> data) {
```
It can support any type of data (String, Number, Boolean).

Reply:
Done